tests/UNMOCKED_end2end_tests/test_download_large_files.py

```diff
@@ -0,0 +1,16 @@
+import time
+
+import lightly
+lightly.api.utils.RETRY_MAX_RETRIES = 1
+
+from lightly.api.download import download_image
+
+url_33MB = "https://cdn.eso.org/images/original/potw1130a.tif"
+url_5MB = "https://cdn.eso.org/images/large/potw1130a.jpg"
+url_1_5MB = "https://cdn.eso.org/images/publicationjpg/potw1130a.jpg"
+
+start = time.time()
+img = download_image(url_5MB)
```

Comment:
did this test fail before your changes?

Reply:
No, it did not. However, for some implementations I tried, the unittests worked and the actual download from a url failed. Thus I added this test.

Reply:
Currently, the unmocked tests fail because of https://lightly-ai.slack.com/archives/C014FJT529M/p1652273127740529

Reply:
Can you fix this in this PR or is it a bigger problem?

Reply:
I am trying, the fix is the lates commit, the unmocked test is here: https://github.com/lightly-ai/lightly/actions/runs/2307560569